RATapi/utils/enums.py

```diff
@@ -68,60 +101,101 @@ def _missing_(cls, value: Union[int, str]):
         return super()._missing_(value)
 
     def __int__(self):
+        """Convert the DE strategy to its hardcoded index in the internal code."""
         # as RAT core expects strategy as an integer
         return list(Strategies).index(self) + 1
 
 
 class BoundHandling(RATEnum):
-    """Defines the available options for bound handling"""
+    """The available options for bound handling in DREAM."""
 
     Off = "off"
+    """Allow points to be sampled out of the parameter bounds."""
+
     Reflect = "reflect"
+    """If a step passes a boundary, reflect it back across the boundary."""
+
     Bound = "bound"
+    """If a step passes a boundary, set it equal to the nearest point on the boundary."""
+
     Fold = "fold"
+    """Treat the boundary as periodic and ‘wrap the step around’ to the other side of the space."""
 
 
 # Models
 class TypeOptions(RATEnum):
+    """The types of signal (``Background`` and ``Resolution``)."""
+
     Constant = "constant"
+    """A uniform constant signal given by a parameter."""
+
     Data = "data"
+    """A signal given in the dataset for the contrast."""
```

Comment:
It's only the dataset for the contrast for a resolution, it's an additional dataset for a background. Maybe try: "A signal for each q value given in a dataset "